dev-prefs.json

```diff
@@ -2,6 +2,8 @@
   "dom.ipc.processCount": 1,
   "javascript.options.strict": false,
   "xpinstall.signatures.required": false,
+  "extensions.logging.enabled": true,
   "extensions.@activity-streams.sdk.console.logLevel": "info",
-  "extensions.@activity-streams.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
+  "extensions.@activity-streams.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream",
+  "extensions.@activity-stream-shield-study.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
```

Comment:
@ncloudioj Oh I see this is dev prefs, for the production one do we send it to the regular production activity stream telemetry endpoint and then separate the shield data using an infernyx job?